Backend/dev/migrations-read-only/rider-app/station.sql

```diff
@@ -16,4 +16,10 @@ ALTER TABLE atlas_app.station ADD PRIMARY KEY ( id);
 
 ------- SQL updates -------
 
-ALTER TABLE atlas_app.station ALTER COLUMN merchant_operating_city_id SET NOT NULL;
\ No newline at end of file
+ALTER TABLE atlas_app.station ALTER COLUMN merchant_operating_city_id SET NOT NULL;
+
+
+------- SQL updates -------
+
+ALTER TABLE atlas_app.station ADD COLUMN gmm_station_name text ;
```

Comment:
Let's create this as a separate table linking `partner_org` and the `station` table, instead of making this specific to GMM